app/src/components/account/UserMenu.vue

```diff
@@ -173,8 +173,8 @@ export default {
         networkId: address.networkId,
       })
     },
-    signOut() {
-      this.$store.dispatch(`signOut`, this.network)
+    signOut(address) {
+      this.$store.dispatch(`signOut`, { address, networkId: this.network })
```

Comment:
why do we need to sign the user out of a network here? this button signs the user out of the account, doesn't it?

Reply:
True... good question. Probably bad logic & need to fix it